src/runtime/job.cpp

```diff
@@ -1396,7 +1448,7 @@ static PRIMFN(prim_job_cache) {
   Value *joblist;
   if (reuse.found && !jobtable->imp->check) {
     Job *jobp = Job::claim(runtime.heap, jobtable->imp->db, dir, dir, stdin_file, env, cmd, true,
-                           STREAM_ECHO, STREAM_INFO, STREAM_WARNING);
+                           STREAM_ECHO, STREAM_INFO, std::vector<std::string>(), STREAM_WARNING, std::vector<std::string>());
```

Comment:
You can use `{}` for these cases, but per my previous comment, I'd recommend using `""` and passing a string here to delay the parsing.

Reply:
Ah, yeah, that makes sense.  Let me make sure it doesn't conflict with anywhere we already need to have the `vector`, but otherwise that'll make it cleaner.